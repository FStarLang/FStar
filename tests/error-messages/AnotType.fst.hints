[ "\u000bŸ{*Æù4-Bò\u0010OÛ­@g", [] ]