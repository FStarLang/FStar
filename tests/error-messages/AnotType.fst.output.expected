* Info at AnotType.fst(19,5-19,7):
  - Expected failure:
  - Type annotation Prims.int for inductive AnotType.tc is not Type or eqtype,
    or it is eqtype but contains noeq/unopteq qualifiers

* Info at AnotType.fst(27,14-27,16):
  - Expected failure:
  - Expected expression of type Type0 got expression tb of type Type

